content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -194,6 +194,78 @@ For example, if there are 1,253 pods on the cluster and the client wants to rece
 
 Note that the `resourceVersion` of the list remains constant across each request, indicating the server is showing us a consistent snapshot of the pods. Pods that are created, updated, or deleted after version `10245` would not be shown unless the user makes a list request without the `continue` token.  This allows clients to break large requests into smaller chunks and then perform a watch operation on the full set without missing any updates.
 
+## Lists
+
+There are dozens of list types (such as `PodList`, `ServiceList`, and `NodeList`) defined in the Kubernetes API.
+You can get more information about each list type from the [Kubernetes API](https://kubernetes.io/docs/reference/kubernetes-api/) documentation.
+
+When you query the API for a particular type, all items returned by that query are of that type. For example, when you
+ask for a list of services, the list type is shown as `kind: ServiceList` and each `item` in that list represents a service. For example:
+
+```console
+
+GET /api/v1/services
```

Comment:
`kubectl --raw` might be another way to fetch this. Not sure if that helps or not.

Reply:
I was just trying to match the way that content is shown in the rest of the page. I usually use curl commands to query the API after running `kubectl proxy`. So I'm thinking we should leave that unless we want to change the others.

Reply:
Let me know if you disagree.